# TLB Lab Computational Resources

This is a list of software & Databases that has been developed by or passed through the laboratory of Prof Sir Tom L. Blundell over the years.

Collating this list is ongoing and not exhaustive. It is currently maintained by Arian Jamasb (arj39@cam.ac.uk). If your work is not included, and you would like it to be, please reach out to me by email. However, I am not a maintainer of these works - please contact the relevant authors if you have questions about usage.

## Software

### Protein Structure Modelling
MODELLER - 

FUGUE 

JOY -

CODA

CHORAL: a differential geometry approach to the prediction of the cores of protein structures

PROVAT: a tool for Voronoi tessellation analysis of protein structures and complexes

SCORE: predicting the core of protein models

Rappertk: a versatile engine for discrete restraint-based conformational sampling of macromolecules

ARABESQUE: A tool for protein structural comparison using differential geometry and knot theory

PocketPicker: Analysis of ligand binding-sites with shape descriptors.

### Mutation Modelling 
SDM

mCSM

DUET

pkCSM

mCSM-lig: quantifying the effects of mutations on protein-small molecule affinity in genetic disease and emergence of drug resistance

Ulla: a program for calculating environment-specific amino acid substitution tables

### Sequence Analysis Tools

XSuLT: a web server for structural annotation and representation of sequence-structure alignments

### Docking & Virtual Screening
pyDock

USRCAT: real-time ultrafast shape recognition with pharmacophoric constraints

Arpeggio



## Databases
HOMSTRAD -

CAMPASS

A database of globular protein structural domains: clustering of representative family members into similar folds

CREDO: a protein–ligand interaction database for drug discovery

Platinum: a database of experimentally measured effects of mutations on structurally defined protein–ligand complexes

BIPA: a database for protein–nucleic acid interaction in 3D structures

Comprehensive, atomic-level characterization of structurally characterized protein-protein interactions: the PICCOLO database

TIMBAL v2: update of a database holding small molecules modulating protein–protein interactions

Genome3D

CHOPIN: a web resource for the structural and functional proteome of Mycobacterium tuberculosis

DDBASE2. 0: updated domain database with improved identification of structural domains

SInCRe—structural interactome computational resource for Mycobacterium tuberculosis

ProCarbDB: a database of carbohydrate-binding proteins

TIBLE: a web-based, freely accessible resource for small-molecule binding data for mycobacterial species

Mabellini: a genome-wide database for understanding the structural proteome and evaluating prospective antimicrobial targets of the emerging pathogen Mycobacterium abscessus

XSuLT: a web server for structural annotation and representation of sequence-structure alignments

